File: 391/a.java
import java.util.Scanner;

public class a {

    public static void main(final String[] args) {
        String code = new Scanner(System.in).nextLine();
        int seq = 1, ans = 0;
        char prev = 'Z';
        for (char chr: code.toCharArray()) {
            if (prev == chr) {
                seq++;
            } else {
                //System.out.println(chr + ", lose " + seq);
                if (seq % 2 == 0) {
                    ans++;
                }
                seq = 1;
            }
            prev = chr;
        }
        if (seq % 2 == 0) {
            ans++;
        }
        System.out.println(ans);
    }
}

File: 391/a.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
  int len = 1, ans = 0;
  char prev = 'Z';
  string code;
  cin >> code;

  for (int i = 0, n = code.length(); i < n; ++i) {
    if (code[i] == prev) {
      ++len;
    } else {
      if (len % 2 == 0) {
        ++ans;
      }

      len = 1;
    }

    prev = code[i];
  }

  if (len % 2 == 0) {
    ++ans;
  }

  cout << ans << endl;
  return 0;
}
